packages/react-reconciler/src/__tests__/ReactSuspenseEffectsSemantics-test.js

```diff
@@ -1980,7 +1980,6 @@ describe('ReactSuspenseEffectsSemantics', () => {
 
           // Destroy layout and passive effects in the errored tree.
           'App destroy layout',
-          'ThrowsInWillUnmount componentWillUnmount',
```

Comment:
I think this was a bug since it was already earlier in the list. So removing it seems correct to me.